.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,122 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
You should not commit any file to a repository under the folder .idea (these are configuration files only relevant for your localenvironment)